### PR TITLE
Add Planned Enhancement: Specific Index error message

### LINKED ISSUE #211: [PE-D][Tester E] Error result unclear for enrol feature
Just a very small detail, but for the enrol feature, when one of the parameters of m and e is negative, the result display does not show which is the parameter at fault.

-------------
Labels: `type.FunctionalityBug` `severity.VeryLow`
original: prawnzyy/ped#4

### PR DESCRIPTION
**Pull request by SelwynAng**
Closes #211 

- Added Planned Enhancement to allow for more specific index error message

### CODE REVIEW

File: docs/DeveloperGuide.md [hunk 1]
```diff
@@ -421,6 +421,20 @@ when editing the tags of the member/event:
 3. Edit current list of tags attached to the member/event
 4. Delete specific tags from the current list of tags attached to the member/event
 
+### Provide more specific index error messages to the user
+**Current Implementation**
```
Comment (andrechuakj) [inline docs/DeveloperGuide.md]: Should we add a colon here to be consistent with the other sub-headings?
```suggestion
**Current Implementation:**
```

Comment (andrechuakj) [reply docs/DeveloperGuide.md]: Nit picking here, but there are some inconsistencies in whether the colon is bolded or not in other subheadings as well, maybe we should just stick to bolding the colon?

**Review by andrechuakj — COMMENTED** [review]
Just a nit, the rest LGTM

File: docs/DeveloperGuide.md (hunk 1 above)
Comment (SelwynAng) [reply docs/DeveloperGuide.md]: Okay, let's bold all colons!

**Comment by andrechuakj** [comment]
LGTM

**Review by wr1159 — APPROVED** [review]
LGTM

**Pull request merged**